static/script/devices/mediaplayer/samsung_streaming.js

```diff
@@ -386,6 +379,16 @@ require.def(
             getSeekableRange: function () {
                 return this._range;
```

Comment:
This is handled the same as in samsung_maple.js module.
_range is set to undefined on _wipe() or toStopped().